src/variational/bijectors.jl

```diff
@@ -0,0 +1,72 @@
+
+# TODO: Move to Bijectors.jl if we find further use for this.
+"""
+    wrap_in_vec_reshape(f, in_size)
+
+Wraps a bijector `f` such that it operates on vectors of length `prod(in_size)` and produces
+a vector of length `prod(Bijectors.output(f, in_size))`.
+"""
+function wrap_in_vec_reshape(f, in_size)
+    vec_in_length = prod(in_size)
+    reshape_inner = Bijectors.Reshape((vec_in_length,), in_size)
+    out_size = Bijectors.output_size(f, in_size)
+    vec_out_length = prod(out_size)
+    reshape_outer = Bijectors.Reshape(out_size, (vec_out_length,))
+    return reshape_outer ∘ f ∘ reshape_inner
+end
+
+"""
```

Comment:
Please move `Bijectors.bijector(model::DynamicPPL.Model,...)` to DynamicPPL. 

cc @mhauru

Reply:
[Done](https://github.com/TuringLang/DynamicPPL.jl/pull/920)